bukkit/src/main/java/pl/bestguilds/BukkitBestGuilds.java

```diff
@@ -0,0 +1,32 @@
+package pl.bestguilds;
+
+import org.bukkit.plugin.java.JavaPlugin;
+import org.jetbrains.annotations.NotNull;
+import pl.bestguilds.api.BestGuildsAPI;
+import pl.bestguilds.api.guild.GuildManager;
+import pl.bestguilds.api.user.UserManager;
+
+public final class BukkitBestGuilds extends JavaPlugin implements BestGuildsAPI {
+
+  private BestGuildsPlugin plugin;
+
+  @Override
+  public void onEnable() {
+    this.plugin = new BestGuildsPlugin();
```

Comment:
You forgot to setup the [static instance](https://github.com/BestGuilds/BestGuilds/pull/3/files#diff-307b8e5c9a4ddf981c431d6825437426R17) of the API.